libraries/AP_RCProtocol/AP_RCProtocol_config.h

```diff
@@ -61,3 +61,7 @@
 #ifndef AP_RCPROTOCOL_FASTSBUS_ENABLED
 #define AP_RCPROTOCOL_FASTSBUS_ENABLED AP_RCPROTOCOL_BACKEND_DEFAULT_ENABLED && AP_RCPROTOCOL_SBUS_ENABLED
 #endif
+
+#ifndef AP_RCPROTOCOL_MAVLINK_RADIO_ENABLED
+#define AP_RCPROTOCOL_MAVLINK_RADIO_ENABLED AP_RCPROTOCOL_BACKEND_DEFAULT_ENABLED
```

Comment:
Maybe default enable only on 2M flash boards? I haven't looked at flash cost of the feature

Reply:
Done.